docs/source/basics.rst

```diff
@@ -89,7 +89,7 @@ part of any number of such collections.
 
 Benefiting from Git's support for workflows based on decentralized "clones" of
 a repository, DataLad's datasets can be (re-)published to a new location
-without loosing the connection between the "original" and the new "copy". This
+without losing the connection between the "original" and the new "copy". This
```

Comment:
duplicate of a more thorough https://github.com/datalad/datalad/pull/7271/files#diff-b92c955d6722cda596b80f393d13e7b0ee91eb17a99863114ec364a436079f13R92 .